ros/src/computing/perception/detection/vision_detector/packages/vision_yolo2_detect/src/vision_yolo2_detect.cpp

```diff
@@ -0,0 +1,303 @@
+#include <ros/ros.h>
```

Comment:
Yolo2 node does not exist anymore. This should be removed.

Reply:
Done.